docs/casestudies/adopters.md

```diff
@@ -12,13 +12,14 @@ To join this list, please follow [these instructions](https://github.com/karmada
-| Youzan                    | https://www.youzan.com/     | Manage both Stateless and Stateful on-line workloads across multiple clusters           | TBD                                                                           |
\ No newline at end of file
+| Organization/Company Name | Website                     | Usage Scenario                                                                                           | CaseStudy                                                                     |
+|---------------------------|-----------------------------|----------------------------------------------------------------------------------------------------------|-------------------------------------------------------------------------------|
+| Cibfintech.com            | https://www.cibfintech.com/ | Build a more resilient and elastic hybrid-multi-cloud platform using Karmada                             | TBD                                                                           |
+| DaoCloud.io               | https://www.daocloud.io/    | Build a more resilient and elastic hybrid-multi-cloud platform using Karmada                             | TBD                                                                           |
+| Ecloud                    | https://ecloud.10086.cn/    | Use Karmada to build a steady and elastic multi-cluster platform for managing workloads                  | TBD                                                                           |
+| Hurricane Engines         | /                           | Use Karmada to build multi-cloud PaaS platform with multi-vendor and multi-region                        | [Karmada in AIML INSTITUTE](ci123.md)                                         |
+| Trip.com                  | https://www.trip.com/       | Build a more resilient and elastic hybrid-multi-cloud platform using Karmada                             | TBD                                                                           |
+| VIPKID                    | https://www.vipkid.com/     | Use Karmada to build multi-cluster platform with heterogeneous resources and clusters                    | [Building a PaaS Platform with Karmada to Run Containers --VIPKID](vipkid.md) |
+| Vivo                      | https://www.vivo.com/       | Build a more resilient and elastic hybrid-multi-cloud platform using Karmada                             | TBD                                                                           |
+| Youzan                    | https://www.youzan.com/     | Manage both Stateless and Stateful on-line workloads across multiple clusters                            | TBD                                                                           |
+| Unicom Digital Tech       | http://www.chinaunicom.com.cn| Use Karmada to build a intelligent and distributed container platform with multi-policy and multi-region | TBD                                                                           |
```

Comment:
Given all organizations are sorted alphabetically, could you please add your item before `VIPKID`?

Reply:
I'm sorry for my careless ,i updated it please check it again